src/stats.c

```diff
@@ -1072,11 +1072,14 @@ static UInt ExecIntrStat(Stat stat)
 
     /* and now for something completely different                          */
 #ifdef USE_GASMAN
-    if ( SyStorOverrun != 0 ) {
-      SyStorOverrun = 0; /* reset */
-      ErrorReturnVoid(
-  "reached the pre-set memory limit\n(change it with the -o command line option)",
-        0, 0, "you can 'return;'" );
+    if (SyStorOverrun != SY_STOR_OVERRUN_CLEAR) {
+        SyStorEnum value = SyStorOverrun;
+        SyStorOverrun = SY_STOR_OVERRUN_CLEAR; /* reset */
+        if (value == SY_STOR_OVERRUN_TO_REPORT) {
+            ErrorReturnVoid("reached the pre-set memory limit\n(change it "
```

Comment:
Can we do this, as in: does `clang-format` accept it?
```suggestion
            ErrorReturnVoid("reached the pre-set memory limit\n"
```